desktop/NodeManager.ts

```diff
@@ -459,7 +459,8 @@ class NodeManager {
     if (!this.nodeProcess) return;
     try {
       // Request Node shutdown
-      await this.nodeService.shutdown();
+      this.nodeProcess.kill('SIGKILL');
```

Comment:
Wait. It should not be so rough.
Send `SIGTERM` instead.

```suggestion
      this.nodeProcess.kill('SIGTERM');
```